<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanningHospitalBedProfileDialog</class>
 <widget class="QDialog" name="PlanningHospitalBedProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Планирование загруженности коечного фонда</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="chkDate">
       <property name="text">
        <string>Период с</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtBegDate"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>по</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtEndDate"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="chkOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="chkProfileBed">
     <property name="text">
      <string>Профиль койки</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="btnFill">
     <property name="text">
      <string>Сгенерировать список подразделений</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="CInDocTableView" name="tblItems"/>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="COrgStructureHospitalBedsComboBox" name="cmbOrgStructure">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="CRBComboBox" name="cmbHospitalBedProfile">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureHospitalBedsComboBox</class>
   <extends>QComboBox</extends>
   <header>OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chkDate</tabstop>
  <tabstop>chkOrgStructure</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>chkProfileBed</tabstop>
  <tabstop>cmbHospitalBedProfile</tabstop>
  <tabstop>tblItems</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlanningHospitalBedProfileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlanningHospitalBedProfileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnFill</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
